src/test/paths.ts

```diff
@@ -3,6 +3,64 @@ import {isSameUri, convertClientPathToDebugger, convertDebuggerPathToClient} fro
 import * as assert from 'assert';
 
 describe('paths', () => {
+    let undef: string;
+
+    const pathMapping = {
```

Comment:
I prefer to define the test cases through mocha's `it` DSL over having a config object, it's a lot easier to debug if the values are always inside the test